build/package/windows/nri-386-installer/Product.wxs

```diff
@@ -0,0 +1,99 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <?define Version = "$(var.IntegrationVersion)" ?>
+    <?define ProductId = "*" ?>
+    <?define UpgradeCode = "{1a921296-1d82-4014-83ad-9d0c2f9c9bef}" ?>
```

Comment:
This should be unique , could you regenerate this code? windows uses this in order to identify an installed package .

Reply:
ok